Client/Frontend/Strings.swift

```diff
@@ -341,6 +341,8 @@ extension String {
                     public static let FxAmethystWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.AmethystWallpaper.v99", value: "Firefox wallpaper, amethyst pattern.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string for the amethyst firefox wallpaper.", lastUpdated: .v99)
                     public static let FxSunriseWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.SunriseWallpaper.v99", value: "Firefox wallpaper, sunrise pattern.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the sunrise firefox wallpaper.", lastUpdated: .v99)
                     public static let FxCeruleanWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.CeruleanWallpaper.v99", value: "Firefox wallpaper, cerulean pattern.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the cerulean firefox wallpaper.", lastUpdated: .v99)
+                    public static let FxBeachHillsWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.BeachHillsWallpaper.v100", value: "Firefox wallpaper, beach hills pattern.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the beach hills firefox wallpaper.", lastUpdated: .v100)
+                    public static let FxTwilightHillsWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.TwilightHillsWallpaper.v100", value: "Firefox wallpaper, twilight hills pattern.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the twilight hills firefox wallpaper.", lastUpdated: .v100)
```

Comment:
Export for v100 already happened, should there be a new export? Otherwise let's tag this as v101

Reply:
Yeah, I'll have to export today. It was a last minute addition, which is why it was tagged 100. We're gonna see how much we get translated.